### PR TITLE
Sio esx ips

### PR DESCRIPTION
**Pull request by kushagra-mahajan**

### CODE REVIEW

File: dne-paqx-web/src/test/java/com/dell/cpsd/paqx/dne/rest/NodeExpansionWebApplicationTest.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ public void setUp()
         request.setIdracIpAddress("4.4.4.4");
         request.setIdracSubnetMask("5.55.5.5");
         request.setEsxiManagementSubnetMask("6.66.6.6");
-        request.setScaleIoSvmData1SubnetMask("7.7.7.7");
+        request.setScaleIoDataSvmSubnetMask("7.7.7.7");
```
Comment (gavin-scott) [inline dne-paqx-web/src/test/java/com/dell/cpsd/paqx/dne/rest/NodeExpansionWebApplicationTest.java]: Subnet should be per data network (data1, data2) not per location (svm vs vmk). So I'd expect something like `scaleIoData1SubnetMask` and `scaleIoData2SubnetMask`.

File: dne-paqx/pom.xml [hunk 2]
```diff
@@ -232,6 +232,10 @@
             <artifactId>hibernate-jpa-2.1-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
```
Comment (gavin-scott) [inline dne-paqx/pom.xml]: Still not sure whether this is going to cause issues at some point. But I was playing with it today and it's pretty nice and on the approved legal list so I suppose I'm good to give it a try. Had to install the lombok plugin for intellij and make sure annotation processing was turned on to get it to work in Intellij, but after that worked well.

File: dne-paqx-web/src/test/java/com/dell/cpsd/paqx/dne/rest/NodeExpansionWebApplicationTest.java (hunk 1 above)
Comment (kushagra-mahajan) [reply dne-paqx-web/src/test/java/com/dell/cpsd/paqx/dne/rest/NodeExpansionWebApplicationTest.java]: @gavin-scott Sure @murphk24 and I will make the necessary changes. Now we will take 2 subnet masks for both sioData1 and data2 SVM IPs and one subnet mask for esx ips? IS that right?

File: dne-paqx/pom.xml (hunk 2 above)
Comment (kushagra-mahajan) [reply dne-paqx/pom.xml]: Perfect was going to drop an email with instructions after merging this PR.

**Pull request merged**